<launch>
<!--Launch file for running the multirobot simulation on gazebo-->   
    <arg name="robot_name" default=""/>
    <arg name="exploation_points" default=""/>

    <node pkg="road_based_explorer" type="new_filter_single_v8.py" name="filter" output="screen">

      <param name="map_topic" type="string" value="/$(arg robot_name)/move_base_node/global_costmap/costmap"/>
      <param name="goals_topic" type="string" value="/$(arg robot_name)/detected_points"/>
      <param name="rate" type="int" value="100"/>
      <param name="robot_frame" type="string" value="$(arg robot_name)/map"/>
      <param name="robot_name" type="string" value="$(arg robot_name)"/>
      <param name="exploration_points" value="$(arg exploation_points)"/>
      <param name="little_gain_distance" type="double" value="2"/>

	  </node>


</launch>